src/GovUK.Education.ExploreEducationStatistics.Data.Processor/Services/ProcessorService.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace GovUK.Education.ExploreEducationStatistics.Data.Processor.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using GovUk.Education.ExploreEducationStatistics.Data.Processor.Models;
+    using GovUk.Education.ExploreEducationStatistics.Data.Processor.Services;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.Blob;
+
+    public class ProcessorService : IProcessorService
```

Comment:
The reason I didn't refer to it as an upload service is because we already have an upload to blob step in the service & we already have an importer project so that's the best I could come up with.